ntbs-service/wwwroot/css/search.scss

```diff
@@ -60,14 +60,30 @@
     width: 160px;
 }
 
-.search-header {
-    margin-bottom: 12px;
+.search-label--long {
+    height: 100%;
+    margin: auto 0;
+    width: 180px;

```

Comment:
Is it safe for these labels/legends to be fixed width? Does it still look ok at 200%/400% zoom?

Reply:
Yeah these are only kind of fixed width, they will actually be shrunk by the flexbox to try to fit content in before wrapping, so large zoom/narrow screens work fine with labels that use this because it isn't actually fixed. This is was there before I started work on this.

This is more of a problem with a fieldset legend, which in most browsers do not correctly follow the flexbox rules, so it is a genuinely fixed width. The only component that causes issues with this was the "Sex" radios on the search page, between widths of ~650-700px. That is why I switched the media queries on the search page to switch into narrow media mode at "desktop" width (769px) rather than "tablet" width (641px). In narrow screen mode, there is a single item per line so this is no issue (unless the screen is 180px wide or less, which is so very narrow I don't think it will actually cause a problem). With that media query, I tried a large range of widths and zooms and found no other issues. In short, this is tricky because fieldsets and legends don't play nice together, but I've tested it a load and I think it works properly